packages/edit-post/src/components/keyboard-shortcuts/index.js

```diff
@@ -56,6 +56,7 @@ class EditorModeKeyboardShortcuts extends Component {
 export default compose( [
 	withSelect( ( select ) => ( {
 		isRichEditingEnabled: select( 'core/editor' ).getEditorSettings().richEditingEnabled,
```

Comment:
`isRichEditingEnabled` and `isCodeEditingEnabled` can be combined into one flag to make it all simpler:
```js
const { richEditingEnabled, codeEditingEnabled } = select( 'core/editor' ).getEditorSettings();

return {
    isEnabled: richEditingEnabled && codeEditingEnabled,
    ...
};
```